README.md

```diff
@@ -8,7 +8,7 @@ This repository is intended to be used in conjunction with other parts of the te
 library (standard, os, grid...) to actually configure a system. To help putting together
 everything needed in a way directly usable in a real configuration, it is recommended to
 follow the procedure describe at 
-https://trac.lal.in2p3.fr/Quattor/wiki/Download/QWGTemplates/Install#Upgradinganexistingrepository.
+http://www.quattor.org/documentation/2014/06/06/how-to-use-template-library.html#how-to-get-it?.
```

Comment:
Is the ? at the end of the URL necessary? it looks a bit odd. I think it would also look a bit better if the text was: "it is recommended to follow this `[procedure](...link text...)".`